docs/extras/modules/model_io/output_parsers/xml.ipynb

```diff
@@ -0,0 +1,178 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "181b5b6d",
+   "metadata": {},
+   "source": [
+    "# XML parser\n",
+    "This output parser allows users to obtain results from LLM in the popular XML format. \n",
+    "\n",
+    "Keep in mind that large language models are leaky abstractions! You'll have to use an LLM with sufficient capacity to generate well-formed XML."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "3b10fc55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.prompts import PromptTemplate\n",
+    "from langchain.llms import OpenAI\n",
+    "from langchain.output_parsers import XMLOutputParser"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "909161d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Can you start with a simple example based on something like:
https://docs.anthropic.com/claude/docs/break-tasks-into-subtasks